src/constants.js

```diff
@@ -153,9 +153,24 @@ export const reportRowKeys = ['display_name', 'os', 'performance_utilization.cpu
 
 export const percentageKeys = ['performance_utilization.cpu', 'performance_utilization.memory', 'idling_time'];
 
-export const PDF_REPORT_PER_PAGE = {
-    systemsReportFirstPage: 18,
-    systemsReportRestPages: 22
+export const PDF_RECORDS_PER_PAGE = {
+    firstPageCount: 16,
+    otherPageCount: 22
```

Comment:
@kgaikwad the reason I updated this count for first page is because it was affecting formatting if we add all the filters (state, name) as previously we were using \t and now we put it on new line and hence updated the count for first page.